scripts/globals/mobskills/flying_hip_press.lua

```diff
@@ -10,18 +10,25 @@
 require("scripts/settings/main")
 require("scripts/globals/status")
 require("scripts/globals/mobskills")
-
 -----------------------------------
-local mobskill_object = {}
+local newtonID = require("scripts/zones/Newton_Movalpolos/IDs")
+-----------------------------------
 
+local mobskill_object = {}
 
 mobskill_object.onMobSkillCheck = function(target, mob, skill)
     return 0
 end
 
 mobskill_object.onMobWeaponSkill = function(target, mob, skill)
+    local bonus = 300
+
+     -- Bugbear Matman has stronger Flying Hip Press
+    if mob:getID() == newtonID.mob.BUGBEAR_MATMAN then
+        bonus = 700
+    end
```

Comment:
I really dont like this method of adjusting damage.
What I mean is this:
If something is modifying the TP move dmg, it should be handled via modifier IMO.

thoughts?

Reply:
the pattern we've been using since dsp is to conditional on `getPool()` since mob ID can (albeit rarely) change and we'd wind up with a lot of unique one time used modifiers making it messy.

example: https://github.com/LandSandBoat/server/blob/3dfe1f76e9a5c4fe4392de6ffb77ea2bdc2fb4ef/scripts/globals/mobskills/restoral.lua#L22

Reply:
In general, SE _usually but not always_ has a 2nd copy of  the mobskill in the dats that w ca rename to make alternates with. A capture  ill ell us if is te same IID the normal obs use or if the NM has such an alternate on retail. In some cases the difference really comes down  to the NM simply being that much stronger than the normal mobs making its TP skill stronger but in most cases they just devote a 2nd dat entry to it. Past iteration of the project have not implemented most of that and have been doing conditional branches for most skills instead. If id duplicate ID is used on retail, I would readily accept that in a pr. else I'd go for getPool method.

Reply:
Wel, I'll have to look into this for quite a few NMs I've corrected on Wings. This has been my goto method for introducing different mechanics that NMs have for normal TP moves (Cemetery Cherry comes to mind). I'll ask for a capture.

Reply:
> for introducing different mechanics that NMs have for normal TP moves 

Any that are only a power change, go for getPool anyway if they animate identiclaly. getPool is safe. more complex logic changes you'll want that separate scrip if the way SE did them allows for it.

Reply:
I received a capture for this NM, and its the same ID for both skill and animation. So I've adjusted the line to use mob:getPool() instead.

I am running into a bit of an issue with Flying Hip Press actually. It seems the `bonus` variable is actually a cap on the amount of damage a breath move can do, and uncapped this ability is doing 4000~ damage. So while breath moves should do less damage the lower the mob gets, its going from 4000 down to 1000 but since I've set the cap to 700, you will see zero change to the damage flying hip press does all the way down until the mob dies. It always does exactly 700 damage with no variance.